apps/v3_api/lib/headers.ex

```diff
@@ -21,7 +21,9 @@ defmodule V3Api.Headers do
 
   @spec api_key_header(header_list, String.t() | nil) :: header_list
   defp api_key_header(headers, nil), do: headers
-  defp api_key_header(headers, <<key::binary>>), do: [{"x-api-key", key} | headers]
+
+  defp api_key_header(headers, <<key::binary>>),
+    do: [{"x-api-key", key}, {"MBTA-Version", "2019-07-01"} | headers]
```

Comment:
This is part of a string of functions in `V3Api.Headers.build/2` that adds various key-value pairs to the header for every request Dotcom makes to the V3 API. This function is where we include `"x-api-key": <our V3 API key>`, and is modified here to also include a new `"MBTA-Version": "2019-07-01"` key-value pair. 

My understanding from [this page of the documentation](https://www.mbta.com/developers/v3-api/versioning) is that this will tell the API to use the `2019-07-01` API version for all requests regardless of whatever version the API key itself might have as default.

> Does it stop all api requests if they have the wrong version specified?

Essentially the opposite - if I try to use an API key attached to a different/wrong version number, it should disregard that and proceed with `2019-07-01`.

Reply:
> Are these changes going to require people to reset up the their environment?

Nah, existing environments will still work the same as before with no changes required.

Reply:
Okay, it might make sense to move this to an environment variable.

Reply:
OK, yeah realizing now the Asana ticket suggests to make the value configurable.. ok. Just pushed a commit that uses a new `V3_API_VERSION` environment variable.